src/data/global.json

```diff
@@ -298,7 +298,7 @@
                 }
             ],
             "copyright": {
-                "title": "© 2020 - Mitwirkende Personen des Open-Source-Projekts für Corona-Warn-App",
+                "title": "© 2020-2021- Mitwirkende Personen des Open-Source-Projekts für Corona-Warn-App",
```

Comment:
Here is a whitespace missing after 2021.

Reply:
Thank you! Fixed in [`0174c6a`](https://github.com/corona-warn-app/cwa-website/pull/1102/commits/0174c6a236c9bfcdcb49f75aad68881fd836a07c).